src/app/index.js

```diff
@@ -40,6 +81,9 @@ angular.module('app', [
 
     // If no states were matched, redirect to home
     $urlRouterProvider.otherwise('home')
+
+    $authProvider.baseUrl = 'http://localhost:3000/api'
```

Comment:
Does this need to be merged before the other pull request? This looks like the old way still